### ISSUE TITLE
Crash on assembly list hover/right-click (FileNotFoundException: Accessibility)

### ISSUE THREAD

**Issue by RichardD2** [icsharpcode/ILSpy#3651]
### Steps to reproduce
1. Open ILSpy v10.0.0.8282-preview2
2. Open a .NET 10.0.2 assembly
3. Hover over the assembly in the assemblies list; or: right-click on the assembly

### Error message shown

```
---------------------------
Sorry, we crashed
---------------------------
System.IO.FileNotFoundException: Could not load file or assembly 'Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.

File name: 'Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'

   at System.Windows.Controls.Primitives.Popup.PopupSecurityHelper.ForceMsaaToUiaBridge(PopupRoot popupRoot)

   at System.Windows.Controls.Primitives.Popup.CreateWindow(Boolean asyncCall)

   at System.Windows.Controls.Primitives.Popup.OnIsOpenChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)

   at System.Windows.FrameworkElement.OnPropertyChanged(DependencyPropertyChangedEventArgs e)

   at System.Windows.DependencyObject.NotifyPropertyChange(DependencyPropertyChangedEventArgs args)

   at System.Windows.DependencyObject.UpdateEffectiveValue(EntryIndex entryIndex, DependencyProperty dp, PropertyMetadata metadata, EffectiveValueEntry oldEntry, EffectiveValueEntry& newEntry, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType)

   at System.Windows.DependencyObject.SetValue(DependencyProperty dp, Object value)

   at System.Windows.Data.BindingOperations.SetBinding(DependencyObject target, DependencyProperty dp, BindingBase binding)

   at System.Windows.Controls.Primitives.Popup.CreateRootPopupInternal(Popup popup, UIElement child, Boolean bindTreatMousePlacementAsBottomProperty)

   at System.Windows.Controls.ToolTip.OnIsOpenChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)

   at System.Windows.FrameworkElement.OnPropertyChanged(DependencyPropertyChangedEventArgs e)

   at System.Windows.DependencyObject.NotifyPropertyChange(DependencyPropertyChangedEventArgs args)

   at System.Windows.DependencyObject.UpdateEffectiveValue(EntryIndex entryIndex, DependencyProperty dp, PropertyMetadata metadata, EffectiveValueEntry oldEntry, EffectiveValueEntry& newEntry, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType)

   at System.Windows.DependencyObject.SetValue(DependencyProperty dp, Boolean value)

   at System.Windows.Controls.PopupControlService.ShowToolTip(DependencyObject o, Boolean fromKeyboard)

   at System.Windows.Threading.DispatcherTimer.FireTick()

   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)

   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

### Details
* Product in use: ILSpy
* Version in use: 10.0.0.8282-preview2+ceed812e3d81b2ed9a845d289640cf96a9563e2b
* .NET version: 10.0.2-servicing.25612.105+44525024595742ebe09023abe709df51de65009b
* Sample assembly: `System.Private.CoreLib` (10.0.0.0, .NETCoreApp, v10.0)
  * `C:\Program Files\dotnet\shared\Microsoft.NETCore.App\10.0.2\System.Private.CoreLib.dll`
  * `System.Private.CoreLib, Version=10.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e`

**Comment by RichardD2** [comment]
The error also now seems to trigger when I click on any menu item. It started after installing the January updates for Windows 11 25H2.

**Comment by RichardD2** [comment]
Closing the app produced a different error:

```
---------------------------
Sorry, we crashed
---------------------------
System.DllNotFoundException: Unable to load DLL 'vcruntime140_cor3.dll' or one of its dependencies: The specified module could not be found. (0x8007007E)

   at __std_type_info_destroy_list(__type_info_node*)

   at __scrt_uninitialize_type_info()

   at _app_exit_callback()

   at <CrtImplementationDetails>.ModuleUninitializer.SingletonDomainUnload(Object source, EventArgs arguments)
```

**Comment by RichardD2** [comment]
After restarting the app, the error seems to have gone away.

**Comment by RichardD2** [comment]
All seems OK today. Assume this was a temporary glitch in the matrix.
